frontend/app/views/fragments/page/elevatedCloserGuardianBody.scala.html

```diff
@@ -0,0 +1,51 @@
+@import model.OrientatedImages
+@import views.support.Asset
+@import configuration.Videos
+
+<div class="l-constrained elevated-closer-guardian-body">
+
+    <div class="elevated-closer-guardian-body__title u-content-width--left">
+        <h1 class="elevated-closer-guardian-body__title__h1">Supporters get closer</h1>
+        <h1 class="elevated-closer-guardian-body__title__h1">to the Guardian</h1>
+    </div>
+
+    <div class="elevated-closer-guardian-body__col elevated-closer-guardian-body__video u-content-width--left">
+        <div class="video-preview video-preview--plain">
+            <div class="elevated-preview__video">
+            @fragments.media.videoPlayer(Videos.membershipExplained)
+            </div>
+        </div>
+    </div><div class="elevated-closer-guardian-body__col elevated-closer-guardian-body__text u-content-width--right">
+        <div class="elevated-closer-guardian-body__content">
+            <div class="elevated-closer-guardian-body__list">
+                <p class="elevated-closer-guardian-body__list_p u-responsive-p">As a Guardian Supporter, you’ll enjoy a number of benefits, including:</p>
+                <ul class="elevated-closer-guardian-body__ul">
```

Comment:
How would a good BEM strategy  be? I want to apply a style to the paragraph inside the list div.